### PR TITLE
Jetty 10 - #8591: Indicate units of HttpClient properties

### LINKED ISSUE #8591: Indicate units of HttpClient properties
**Jetty version(s)** 11.0.1

**Enhancement Description**

For example, I assume that `HttpClient.setRequestBufferSize()` and `HttpClient.setResponseBufferSize()` refer to a value in bytes but it would be helpful for the Javadoc to state this explicitly. Alternatively, rename the parameter name from say `requestBufferSize` to `requestBufferSizeInBytes`.

### PR DESCRIPTION
**Pull request by cowwoc**

### CODE REVIEW

File: jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ public class HttpClient extends ContainerLifeCycle
     private boolean followRedirects = true;
     private int maxConnectionsPerDestination = 64;
     private int maxRequestsQueuedPerDestination = 1024;
-    private int requestBufferSize = 4096;
+    private int requestBufferSizeInBytes = 4096;
```
Comment (sbordet) [inline jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java]: No need to change the field name.

File: jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java [hunk 2]
```diff
@@ -906,21 +906,21 @@ public void setMaxRequestsQueuedPerDestination(int maxRequestsQueuedPerDestinati
     @ManagedAttribute("The request buffer size")
```
Comment (sbordet) [inline jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java]: Change this description adding `in bytes`.

File: jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java [hunk 3]
```diff
@@ -906,21 +906,21 @@ public void setMaxRequestsQueuedPerDestination(int maxRequestsQueuedPerDestinati
...
     public int getRequestBufferSize()
     {
-        return requestBufferSize;
+        return requestBufferSizeInBytes;
     }
 
     /**
-     * @param requestBufferSize the size of the buffer used to write requests
+     * @param requestBufferSizeInBytes the size of the buffer (in bytes) used to write requests
```
Comment (sbordet) [inline jetty-client/src/main/java/org/eclipse/jetty/client/HttpClient.java]: No need to change the parameter name.
Keep `(in bytes)` in the javadoc.

**Review by sbordet — CHANGES_REQUESTED** [review]

**Comment by cowwoc** [comment]
@sbordet Updated. Try now.

**Review by sbordet — APPROVED** [review]

**Pull request merged**